frontend/src/components/Body/AboutUs/AboutUs.js

```diff
@@ -8,19 +8,7 @@ import MainImage from '../../../images/AboutUs/MainImage.png';
 import Bubbles from "../../../images/AboutUs/Bubble.png";
 
 
-const Container = styled(Box)`
-  position: relative;
-  margin-top: 80px;
-  padding: 0 7% 0 7%;
-
-  @media(max-width: 800px){
-    margin-top: 100px;
-  }
-
-  @media(max-width: 450px){
-    margin-top: 50px;
-  }
-`;
+const Container = styled(Box)``;
 
 const StyledGrid = styled(Grid)`
   @media(max-width: 900px){
```

Comment:
Remove media query, MUI have different media props for same functional. Remove const and add a default "Grid" component. Leave corresponding comments near all containers like "Container for.."